platform/jewel/samples/ide-plugin/src/main/kotlin/org/jetbrains/jewel/samples/ideplugin/JewelDemoToolWindowFactory.kt

```diff
@@ -20,6 +20,8 @@ import org.jetbrains.jewel.samples.ideplugin.releasessample.ReleasesSampleCompos
 @ExperimentalCoroutinesApi
 internal class JewelDemoToolWindowFactory : ToolWindowFactory, DumbAware {
     override fun createToolWindowContent(project: Project, toolWindow: ToolWindow) {
+        System.setProperty("jewel.customPopupRender", "true")
```

Comment:
Why not add a setter to the flags API? So we can avoid the stringly-typed API typical of system props.